### PR TITLE
Help contributors figure out how to test

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

**Comment by paulcsmith** [comment]
Hey @ysbaddaden! Just a quick ping to see if this can be merged

**Comment by ysbaddaden** [comment]
Actually, I don't use chrome (I use Firefox) nor chromedriver and just start the selenium jar, so it would be a very weird advice :confused:

**Comment by paulcsmith** [comment]
Ah ok. Should I go ahead and close this then or would it still be useful to new contributors?

**Comment by bararchy** [comment]
@paulcsmith how about "docker pull" the official image (there are both FF and chrome) , and show a minimal example of `docker run` while exposing the default port

File: README.md [hunk 1]
```diff
@@ -49,18 +48,26 @@ pp count = types.size # => 5
 session.stop
 ```
 
+## Testing Selenium Webdriver Crystal
+
+- Install Chromedriver
```
Comment (Juice10) [inline README.md]: We should probably include a link to the chromedriver homepage: http://chromedriver.chromium.org/home

**Comment by paulcsmith** [comment]
Will close this. Can be reopened by @ysbaddaden if something changes. Thanks for taking a look!

**Pull request closed without merging**